tests/testthat/test-newton_solver.R

```diff
@@ -0,0 +1,44 @@
+# basic test that solver returns numerics within range
+skip("Newton solver has known numerical instability")
+test_that("Newton solver works", {
+  # prepare some data for the solver
+  r0 <- 1.3
+  contact_matrix <- matrix(r0 / 200.0, 2, 2)
+  # because Newton solver fails when the contact matrix is singular
+  contact_matrix <- contact_matrix + runif(prod(dim(contact_matrix)), 0, 0.001)
+  demography_vector <- rep(100.0, 2)
+  psusc <- matrix(1, nrow = 2, ncol = 1)
+  susc <- psusc
+
+  epi_outcome <- solve_final_size_newton(
```

Comment:
Call to `solve_final_size_newton()` which no longer exists, you're probably leaving this file for a later implementation, but just in case you forgot to delete I thought I'd raise it

Reply:
Removed along with file in https://github.com/epiverse-trace/finalsize/pull/42/commits/d46f915eb90b95a19508afeccd73abda6240bd5b